challengeMk2/DataBase/SQLiteDataService.cs

```diff
@@ -8,41 +7,41 @@ namespace ChallengeMk2.DataBase
 {
     public class SQLiteDataService : ILocalDataService
     {
-        SQLiteConnection database;
+        SQLiteAsyncConnection database;
 
-        public void Initialize()
+        public async Task Initialize()
         {
             if (database == null)
             {
-                database = new SQLiteConnection(DatabaseParameters.DbPath, DatabaseParameters.DbFlags);
-                database.CreateTable<StarSystemDbItem>();
+                database = new SQLiteAsyncConnection(DatabaseParameters.DbPath, DatabaseParameters.DbFlags);

```

Comment:
`DatabaseParameters.DbPath`& `DatabaseParameters.DbFlags` are static properties.
We try to avoid static as much as possible. I would prefer to see those parameters passed into the constructor, saved in private fields, then used in the Initialize method.

Reply:
In fact, i can put them directly in the SQLiteDataService class in private fields, they will be used there only. There is no need to use them elsewhere.

Reply:
and remove "DatabaseParameters" class.